docker/entrypoint.sh

```diff
@@ -73,15 +107,34 @@ while true; do
     shift
 done
 
-echo "$APP_ARGS" >/etc/weave/scope-app.args
-echo "$PROBE_ARGS" >/etc/weave/scope-probe.args
+if is_running $WEAVE_CONTAINER_NAME; then
```

Comment:
We're in the container on the weave network now. Couldn't we just check for ethwe?